CMakeLists.txt

```diff
@@ -578,6 +578,8 @@ if (WIN32 OR MINGW)
   add_compile_definitions (_BIND_TO_CURRENT_VCLIBS_VERSION=1 _CRT_SECURE_NO_WARNINGS _CONSOLE)
 endif ()
 
+option (HDF5_MINGW_STATIC_GCC_LIBS "Statically link libgcc/libstdc++" OFF)
```

Comment:
I see the default is off, so I will ask one favor:
For our testing we will need this option set in the config/cmake/cacheinit.cmake file around line 42 (before the external library settings)  just add:
set (HDF5_MINGW_STATIC_GCC_LIBS ON CACHE BOOL "Statically link libgcc/libstdc++" FORCE)
Thank you.

Reply:
ok, done